src/lib/azure/keyvault.test.ts

```diff
@@ -0,0 +1,60 @@
+import uuid from "uuid/v4";
```

Comment:
Thanks for creating tests for components like these 👍🏾 For the sake of reviewers it might be good in the future to create a smaller PRs that have unit test and implementation for components like this. That way each PR has a clear focus

Reply:
Agreed. Added a validation for input arguments and thought unit tests are needed to test it.